django_project/ford3/templates/qualification.html

```diff
@@ -8,77 +9,175 @@
+                            <b>Require a portfolio: </b>{{ requirement.portfolio|yesno:"YES,NO" }}
+                            <br />
+                            <b>Require and APS score: </b>{{ requirement.require_aps_score|yesno:"YES,NO" }}
+                        </div>
+                        <div class="col-md-6">
+                            <b>What does the portfolio require: </b>{{ requirement.portfolio_comment }}
+                        </div>
+                    </div>
+                    <div class="row pb-3">
+                        <div class="col-md-6">
+                            <b>Does qualification require certain subjects: </b>{{ requirement.require_certain_subjects }}
+                        </div>
+                    </div>
+                        <div class="row pb-3">
+                        {% for subject in qualification.entrance_req_subjects_list %}
+                            <div class="col-md-6">
+                                <b>Subject {{ forloop.counter }}: </b>{{ subject.name }} {{ subject.minimum_score }}%
+                            </div>
+                        {% endfor %}
+                        </div>
+                    {% endfor %}
+                </div>
+            </div>
+            <div class="row pt-3 pb-3 nomlr">
+                <div class='col-md-10 edu-underline noplr'>
+                    <h4 class="mt-3">4. Interests and Jobs</h4>
+                </div>
+                <div class='col-md-2'>
+                    <a class="edu-button edu-button-orange" href="#">Edit</a>
+                </div>
+            </div>
+            <div class='row pt-3 pb-3'>
                 <div class='col-md-12'>
+                    <div class="row pb-3">
+                        <div class="col-md-6">
+                            <b>Choose three interests associated to this qualification:</b>
+                            <br />
+                            {% for  interest in qualification.interest_name_list %}
+                                {{  interest }}
+                                {% if forloop.counter < qualification.interest_name_list|length %}, {% endif  %}
+                            {% endfor %}
+                        </div>
+                        <div class="col-md-6">
+                            <b>Choose up to five occupations that this qualification could prepare you for:</b>
+                            <br />
+                            {% for occupation in qualification.occupation_name_list %}
+                                {{  occupation }}
+                                {% if forloop.counter < qualification.occupation_name_list|length %}, {% endif  %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                    <div class="row pb-3">
+                        <div class="col-md-6">
+                            <b>Does the qualification prepare for a critical skills occupation:</b>
```

Comment:
we could use a green check similar to  ☑️if the value is True
and ❌if the value is False
or use checkbox input in read-only